rpc/src/v1/helpers/light_fetch.rs

```diff
@@ -45,15 +45,26 @@ use ethereum_types::{U256, Address};
 use hash::H256;
 use parking_lot::Mutex;
 use fastmap::H256FastMap;
-use transaction::{Action, Transaction as EthTransaction, SignedTransaction, LocalizedTransaction};
+use transaction::{Action, Transaction as EthTransaction, PendingTransaction, SignedTransaction, LocalizedTransaction};
 
 use v1::helpers::{CallRequest as CallRequestHelper, errors, dispatch};
 use v1::types::{BlockNumber, CallRequest, Log, Transaction};
 
+pub use self::dispatch::{LightDispatcher};
```

Comment:
One last thing, this doesn't have to be public so I suggest 

```suggestion
use dispatch::LightDispatcher;
```

Reply:
Thanks, I noticed that `dispatch` already defined, so removed this line at all.